### PR TITLE
[TAS-1455] ✨ Add search API for bookstore

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/util/airtable.ts [hunk 1]
```diff
@@ -143,7 +143,8 @@ export async function queryAirtableForPublication({ query }) {
         type,
         id,
         name,
-        imageUrl,
+        url: `https://${LIKER_LAND_HOSTNAME}/nft/${(id as string).startsWith('col_book') ? 'collection' : 'class'}/${id}`,
```
Comment (nwingt) [inline src/util/airtable.ts]: I think `url` should be a formula field in Airtable

File: src/util/airtable.ts [hunk 2]
```diff
@@ -143,7 +143,8 @@ export async function queryAirtableForPublication({ query }) {
...
+        imageUrl: (imageUrl as string)?.replace('ar://', 'https://arweave.net/'),
```
Comment (nwingt) [inline src/util/airtable.ts]: `imageUrl` should be converted to `https://` when creating a record

You can also use [`parseImageURLFromMetadata()`](https://github.com/likecoin/likecoin-api-public/blob/3b8b45e1a0f341585fa4221d3641a0c1bdc1a364/src/util/api/likernft/metadata.ts#L49)

File: src/util/airtable.ts [hunk 3]
```diff
@@ -94,4 +94,68 @@ export async function createAirtablePublicationRecord({
...
+        maxPrice,
+        description,
+        iscnId,
+      }));
+    return result;
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error(err);
+  }
+  return null;
```
Comment (nwingt) [inline src/util/airtable.ts:158]: Should we return an empty array?

Comment (williamchong) [reply src/util/airtable.ts:158]: null to tell it is an error case instead of real empty result

**Review by nwingt — APPROVED** [review]

**Pull request merged**